src/alto-ui/Form/TextField/TextField.scss

```diff
@@ -60,6 +60,7 @@
   flex: 1;
   flex-basis: auto;
   min-width: 0;
+  line-height: 1;
```

Comment:
did you tried with text with letter bellow the line like: p,g, j, q... they are not cut ?

Reply:
![image](https://user-images.githubusercontent.com/5168044/61847387-c12aed00-aeaa-11e9-99cd-808d9e4108b0.png)